code/area.dm

```diff
@@ -2821,11 +2821,10 @@ area/station/security/visitation
 
 /area/station/turret_protected/Entered(O)
 	..()
-	if (isliving(O))
-		if(!issilicon(O))
-			if (motioncamera)
-				motioncamera.newTarget(O)
-			popUpTurrets()
+	if (isliving(O) && !issilicon(O) && !isintangible(O))
```

Comment:
it'd be better to do an early return here, so you can use ORs and they'll short circuit

Reply:
fixed